docs/guides/in-product-word-list.mdx

```diff
@@ -0,0 +1,522 @@
+
+Use **remove** when hiding information from the current view but not permanently
+erasing it.
+
+| **✅ Do **                        | **❌ Don't**                      |
+| --------------------------------- | --------------------------------- |
+| Remove users over your plan limit | Delete users over your plan limit |
+| Remove tax rate                   | Delete tax rate from group rate   |
+
+Never use **destroy** unless Wreck-it Ralph is making a cameo.
+
+<br />
+
+### enable / disable / turn on / turn off / connect / disconnect
+
+When discussing Jobber features, use **turn on** and **turn off** for
+controlling the availability of it on an account.
+
+| **✅ Do **              | **❌ Don't**           |
+| ----------------------- | ---------------------- |
+| Turn on Jobber Payments | Enable Jobber Payments |
+
+When referencing the apps in the App Marketplace, use **connect** and
+**disconnect** for controlling their availability.
+
+| **✅ Do **        | **❌ Don't**      |
+| ----------------- | ----------------- |
+| Connect Mailchimp | Turn on Mailchimp |
+
+<br />
+
+### export / import / download / upload
+
+Use **export** and **import** when data is moving in or out of Jobber and is
+converted into a different format.
+
+| **✅ Do **    | **❌ Don't**    |
+| ------------- | --------------- |
+| Import CSV    | Upload CSV      |
+| Export Report | Download Report |
+
+Use **download** and **upload** when data is being transferred in or out of
+Jobber in the same format.
+
+| **✅ Do **           | **❌ Don't**       |
+| -------------------- | ------------------ |
+| Upload photo         | Import photo       |
+| Download attachments | Export attachments |
+
+<br />
+
+### got it
+
+Use **got it** as a call to action \*\*\*\*when a user can acknowledge a message
+without taking action on it, like in promo cards or onboarding.
+
+<br />
+
+### link / attach / copy / clone / duplicate
+
+Use **link** when the same information is shown elsewhere in Jobber but not
+copied to create a new version.
+
+| **✅ Do**                          | **❌ Don't**                       |
+| ---------------------------------- | ---------------------------------- |
+| Link quote note to related invoice | Copy quote note to related invoice |
+
+Use **copy** when Jobber creates a new version that would have no effect on the
+original when edited.
+
+| **✅ Do**  | **❌ Don't**    |
+| ---------- | --------------- |
+| Copy Quote | Duplicate Quote |
+
+Use **attach** when describing receipts, forms or signatures being added to an
+object. For file selection, use **select a file** for consistency.
+
+| **✅ Do**                                                  | **❌ Don't**                                               |
+| ---------------------------------------------------------- | ---------------------------------------------------------- |
+| Attach a receipt to this expense <br /> [Select a file](#) | Attach a receipt to this expense <br /> [Attach a file](#) |
```

Comment:
Minor issue but could we pass the `(url)` in these examples to point back to their subsection? Every one of these jumps me back to the top of the page with the `(#)`, which I guess is only a problem if I actually click the link, which is kind of on me, but it would be nice if it didn't jump so far!